clang/www/get_started.html

```diff
@@ -67,15 +67,13 @@ <h3 id="buildNix">On Unix-like Systems</h3>
   <li>Build LLVM and Clang:
   <ul>
     <li><tt>cd llvm-project</tt></li>
-    <li><tt>mkdir build</tt> (in-tree build is not supported)</li>
-    <li><tt>cd build</tt></li>
     <li>This builds both LLVM and Clang in release mode. Alternatively, if
         you need a debug build, switch Release to Debug. See
         <a href="https://llvm.org/docs/CMake.html#frequently-used-cmake-variables">frequently used cmake variables</a>
         for more options.
     </li>
-    <li><tt>cmake -DLLVM_ENABLE_PROJECTS=clang -DCMAKE_BUILD_TYPE=Release -G "Unix Makefiles" ../llvm</tt></li>
```

Comment:
I don't necessarily agree that `-S` and `-B` are newer or better than what our documentation currently says. `-G "Unix Makefiles"` is clearly outdated, though, in the presence of `ninja`.

Reply:
@Endilll It saves two lines. Concision helps grokking

Reply:
Yeah, instead of spreading the complexity between three lines, it's now concentrated in one.
I'd expect users to be more familiar with `cd` and `mkdir` than with `-S` and `-B`, so this change would make the docs less accessible.

Reply:
> Is it purely as shorthand for cd and mkdir effectively, or does it do more beyond that?

I don't think it does. In the current documentation, CMake uses the current working directory as binary directory, and take source directory as a unnamed argument. I believe this is a very common way to use CMake. `-S` and `-B` just make binary and source paths more explicit.

Reply:
Ok your call. I amended the patch.